### PR TITLE
[flatbuffers] update to v22.10.26

### LINKED ISSUE #27273: [flatbuffers] update to v22.9.29
Library name: flatbuffers

New version number: [v22.9.29](https://github.com/google/flatbuffers/releases/tag/v22.9.29)

### PR DESCRIPTION
**Pull request by JonLiu1993**
Fix #27273 

Update flatbuffers to the latest version 22.10.26

Note: no feature need to test

### CODE REVIEW

File: ports/flatbuffers/portfile.cmake [hunk 1]
```diff
@@ -3,12 +3,10 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO google/flatbuffers
-    REF v2.0.6
-    SHA512 be631f34064c28d81876bf137c796e9736623cf2cc4f2a05dd45372e7195729c99fad1fa795f8ce71a408756a842edbdc0c3bc714a7cf63203a1de8681d86fb6
+    REF c92e78a9f841a6110ec27180d68d1f7f2afda21d #v2.9.29
```
Comment (ice0) [inline ports/flatbuffers/portfile.cmake:6]: `v2.9.29` -> `v22.9.29`

**Comment by Adela0814** [comment]
@JonLiu1993  Are you fixing CI failures?

**Comment by jdkuki** [comment]
Any update here? As of latest it looks like vcpkg is referencing 2.0.6 which isn't even listed on the git releases page anymore

**Comment by JonLiu1993** [comment]
> Any update here? As of latest it looks like vcpkg is referencing 2.0.6 which isn't even listed on the git releases page anymore

@jdkuki, I'm making time to work around the error locally on ci, I'll fix this pr soon.

**Review by Adela0814 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks!

**Comment by jdkuki** [comment]
Thank you!

**Pull request merged**